components/k8s-model-server/images/Dockerfile.gpu

```diff
@@ -88,9 +88,16 @@ RUN cd /root/serving/tensorflow && ./configure
 RUN cd /root/serving && \
     bazel build -c opt --copt=-mavx --copt=-mavx2 --copt=-mfma --copt=-mfpmath=both --copt=-msse4.2 --config=cuda -k --verbose_failures --crosstool_top=@local_config_cuda//crosstool:toolchain tensorflow_serving/model_servers:tensorflow_model_server
 
-# Add some softlinks for tensorflow model server and CUDA
-RUN ln -s /root/serving/bazel-bin/tensorflow_serving/model_servers/tensorflow_model_server /usr/bin/tensorflow_model_server
+# Make a copy to /usr/bin instead of making softlinks because /root has 700 permissions
+RUN cp /root/serving/bazel-bin/tensorflow_serving/model_servers/tensorflow_model_server /usr/bin/tensorflow_model_server
```

Comment:
FWIW I had a container running from the old image and I verified that if we copy it a new location it still runs; i.e. no linking issues.